inline-gdocs-viewer.php

```diff
@@ -106,6 +107,10 @@ public function addAdminScripts () {
     }
 
     public function addFrontEndScripts () {
+        doAction('igsv_front_end_scripts');
```

Comment:
Isn't it `do_action()`? This would cause a fatal error as PHP attempts to execute an undefined function.